### DISCUSSION TITLE
“parsed IKE_AUTH response” can leaked string text “cert chain, id” ?

### DISCUSSION THREAD

**Discussion by quangm8899** [community/community#122115] [Programming Help]
### Select Topic Area

Question

### Body

thankfully

**Comment by ebndev** [comment]
Hi @quangm8899, we're happy you're here! You are more likely to get a useful response if you are explicit about what your question is, giving a few more details might help someone give you a nudge in the right direction.

**Comment by LashCo** [comment]
Response #1
